cnsctl/cmd/ov/ls.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ov
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"os"
+	"github.com/spf13/cobra"
+)
+
+var datastores, cfgFile string
+var all, long bool
+
+// lsCmd represents the ls command
+var lsCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "List Orphan volumes",
+	Long:  "List Orphan volumes",
+	Run: func(cmd *cobra.Command, args []string) {
+		validateOvFlags()
+		validateLsFlags()
+		// TODO: Add implementation
+	},
+}
+
+func InitLs() {
+	lsCmd.PersistentFlags().StringVarP(&datastores, "datastores", "d", viper.GetString("datastores"), "comma-separated datastore names (alternatively use CNSCTL_DATASTORES env variable)")
+	lsCmd.PersistentFlags().StringVarP(&cfgFile, "kubeconfig", "k", viper.GetString("kubeconfig"), "kubeconfig file (alternatively use CNSCTL_KUBECONFIG env variable)")
```

Comment:
in case if datastore is used by multiple Kubernetes clusters on the same vCenter? do want to supply other k8s cluster's config files as well? when datastore is shared, by only connecting to one k8s cluster, we may not be able to accurately identify orphan volumes.

Reply:
I was thinking of supporting that use case as an improvement to the basic tool. Let me change the comment here that the user can specify more than 1 kubeconfig files.